deploy/general-deploy/kubeturbo.yaml

```diff
@@ -1,30 +1,33 @@
-apiVersion: v1
-kind: Pod
+apiVersion: extensions/v1beta1
+kind: Deployment
 metadata:
   name: kubeturbo
   labels:
-    name: kubeturbo
+    app: kubeturbo
 spec:
-#  uncomment the following line if master is http://127.0.0.1:8080
-#  hostNetwork: true
-  containers:
-  - name: kubeturbo
-#   change version
-    image: vmturbo/kubeturbo:6.0
-    command:
-      - /bin/kubeturbo
-    args:
-      - --v=2
-#     change master url; for kubernetes 1.6+, master parameter can be omitted.
-      - --master=http://1.2.3.4:8080
-      - --turboconfig=/etc/kubeturbo/config
-    volumeMounts:
-    - name: turbo-config
-      mountPath: /etc/kubeturbo
-      readOnly: true
-  volumes:
-  - name: turbo-config
-    hostPath:
-#     change the path to config
-      path: /etc/kubeturbo
-  restartPolicy: Always
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: kubeturbo
+    spec:
+      containers:
+        - name: kubeturbo
+          # Replace the image with desired version
+          image: vmturbo/kubeturbo:6.1dev
```

Comment:
better put the tag as `latest`

Reply:
We don't have `latest` tag. Currently, 6.1dev is used and this example will be updated once we have new versions.